src/main/java/dev/snowdrop/release/App.java

```diff
@@ -170,10 +183,20 @@ public void startRelease(
     
     @CommandLine.Command(name = "list-cves", description = "List CVEs for the specified release or, if not specified, unresolved CVEs")
     public void listCVEs(
+        @CommandLine.Option(names = {"-r", "--release"}, description = "Release the CVE list to github") boolean release,
+        @CommandLine.Option(names = {"-o", "--token"}, description = "Github API token. Required if --release is enabled") String token,
         @CommandLine.Parameters(description = "Release for which to retrieve the CVEs, e.g. 2.2.10", arity = "0..1") String version
     ) throws Throwable {
         final var cves = cveService.listCVEs(Optional.ofNullable(version));
         reportStatus(cves);
+        String mdReport = cveReportSvc.buildMdReport(cves, "Version - " + Optional.ofNullable(version).orElse(" ALL OPEN"));
+        if (release) {
+            if (Optional.ofNullable(token).isPresent()) {
+                git.createGithubIssue(mdReport, "CVE " + (version != null ? "for " + version : "list"), "cve", token, "snowdrop/reports");
+            } else {
+                LOG.error("Cannot release CVE to GitHub. Github API token is required if --release is enabled!");
```

Comment:
Maybe specify which option to use to pass the token?

Reply:
:+1: